src/main/java/org/broadinstitute/hellbender/tools/funcotator/Funcotator.java

```diff
@@ -342,6 +339,23 @@ public void apply(final VariantContext variant, final ReadsContext readsContext,
 
     @Override
     public Object onTraversalSuccess() {
+
+        // If we only saw IGRs, we most likely have a configuration issue.
+        // Make sure the user knows this by making a HUGE stink about it.
+        if ( funcotatorEngine.onlyProducedIGRs() ) {
+            logger.warn("================================================================================");
+            logger.warn("\u001B[43m     _  _  _   __        __               _                   _  _  _           ");
+            logger.warn("    | || || |  \\ \\      / /_ _ _ __ _ __ (_)_ __   __ _      | || || |        ");
+            logger.warn("    | || || |   \\ \\ /\\ / / _` | '__| '_ \\| | '_ \\ / _` |     | || || |     ");
+            logger.warn("    |_||_||_|    \\ \\V V / (_| | |  | | | | | | | | (_| |     |_||_||_|        ");
+            logger.warn("    (_)(_)(_)     \\_/\\_/ \\__,_|_|  |_| |_|_|_| |_|\\__, |     (_)(_)(_)      ");
+            logger.warn("                                                  |___/                         \u001B[0;0m");
+            logger.warn("--------------------------------------------------------------------------------");
+            logger.warn(" Only IGRs were produced for this dataset.  This STRONGLY indicates that this   ");
+            logger.warn(" run was misconfigured.     ");
+            logger.warn(" You MUST check your data sources to make sure they are correct for these data.");
```

Comment:
The mismatch is not between the VCF and reference dictionary, but between the reference dictionary and the data sources dictionary.  This is a by-product of allowing fuzzy-matching between `B37` and `HG19` references (which is very clear at this point was a mistake).

I've added an issue (#5347) to fix the data sources for `B37` and `HG19` so that we can have dedicated data sources for each reference.